atlasdb-config/build.gradle

```diff
@@ -27,4 +27,7 @@ dependencies {
 
     testCompile group: "org.jmock", name: "jmock", version: libVersions.jmock
     testCompile group: 'org.mockito', name: 'mockito-core'
+    testCompile ("com.github.tomakehurst:wiremock:1.57") {
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
```

Comment:
No excludes should be necessary anymore due to the nebula plugin. Try moving the version into versions.props (and use the nicer format (group:, name:)!)